docs/usermanual/userman_admin.rst

```diff
@@ -662,8 +662,10 @@ Most common settings
 ^^^^^^^^^^^^^^^^^^^^
 
 * DEBUG = <True or False> (default = False) -- Enables debug mode. In case you run into technical issues, enable this setting before troubleshooting / reporting.
-* CONTENT_ROOT = "<path to desired content folder>" (default=ka-lite/content)
+* CONTENT_ROOT = "<path to desired content folder>" (default=~/.kalite/content)
   This is the path that KA Lite will use to look for KA Lite video files to play.  Change the path to another local directory to get video files from that directory. NB! Directory has to be writable for the user running the server in order to download videos.
+* ASSESSMENT_ITEMS_DATABASE_PATH = "<path to desired content folder>" (default=~/.kalite/content/assessmentitems.sqlite)
```

Comment:
The problem is that `ASSESSMENT_ITEMS_DATABASE_PATH` is used in `settings.DATABASES` so it can't be changed by apps unless we start figuring out how that pattern would look.. and I'm not optimistic that it's actually possible. So I left it as a top-level setting.

The other settings for assessment items are non-configurable derivatives of CONTENT_ROOT.